doc/debian/jitsi-meet/jitsi-meet.example

```diff
@@ -52,4 +56,28 @@ server {
     location @root_path {
         rewrite ^/(.*)$ / break;
     }
+
+    location ~ ^/([^/?&:'"]+)/config.js$
```

Comment:
Can we have a variable with `^/([^/?&:'"]+)` sow e don't repeat it?

Reply:
No idea :) Need to ask the nginx specialist :) @aaronkvanmeerten WDYT?

Reply:
@saghul It is not recommended:

A: Variables should not be used as template macros. Variables are evaluated in the run-time during the processing of each request, so they are rather costly compared to plain static configuration. Using variables to store static strings is also a bad idea.

Reply:
Sigh. Ok then!